docs/software_requirements/hw_arch_interface.sdoc

```diff
@@ -1,12 +1,12 @@
 [DOCUMENT]
 TITLE: Hardware Architecture Interface
-REQ_PREFIX: ZEP-
+REQ_PREFIX: ZEP-SRS-14
```

Comment:
Prefix should end in a dash so the Requirements are uniquely formatted.  
Maintain the pattern from previous documents.

Reply:
good catch thanks, fixed.